ocs_ci/ocs/precheck.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
```

Comment:
Do we really need a separate file for this function alone? Can we have it moved inside ocs_ci/ocs/node.py?